docker-compose.yml

```diff
@@ -256,8 +309,9 @@ services:
     #       memory: 250M
 
   mssql:
-    profiles: ['all', 'api', 'appeals', 'comment']
+    profiles: ['all', 'api', 'appeals', 'auth', 'comment']
     image: mcr.microsoft.com/azure-sql-edge:latest
+    init: true
```

Comment:
this seems to make stopping the server much faster, not sure what other impact this has, but seems like a nice QOL improvement if nothing else